addons/website/views/snippets.xml

```diff
@@ -1043,6 +1085,33 @@
         </we-select>
     </div>
 
+    <!-- DISPLAY | .s_countdown -->
+    <div data-js="countdown" data-selector=".s_countdown">
+        <we-datetimepicker string="Due Date" data-end-time="" data-unit=""/>
+        <we-select string="What happen at the end?">
+            <we-button data-end-action="nothing">Nothing</we-button>
+            <we-button data-end-action="redirect">Redirect</we-button>
+            <we-button data-end-action="message">Show Message</we-button>
+        </we-select>
+        <we-input string="URL" data-redirect-url="" data-unit="" data-default-value="/my-awesome-page"/>
```

Comment:
This input should suggest website pages (we have an utility method to do that).

Also did you check that the case where an user creates a page with a publishing date and a countdown which ends at the same date ? Won't the page be published one second too late ? Also would this not break SEO to have a link in the DOM leading to a 404 while that page is not published ? (I don't know, this is a real question).

Reply:
- Second option yes
- ? :)

Reply:
- Done
- As discussed, there is no python action involved on a page date publishing, it is just automatically accessible starting from that date (if the page is also published) when reading the `website_published` field which is checking the date + publish state.
I am not sure why but yeah, it is required to wait a bit before landing on the page, 500ms seems enough.